### ISSUE TITLE
Slash Swapper makes clan and friend chat swapped

### ISSUE THREAD

**Issue by jevonsp** [MarbleTurtle/Color-Coordinator#11]
I realize this is a plugin conflict, but is there anyway to detect the plugin and change?

**Comment by ririshi** [comment]
I took a break from the game (again) so I cannot promise any quick updates, but I'm guessing your issue is that this plugin always sends `/`-prefixed messages to FC and `//`-prefixed messages to CC, and you wish for that to be switched if Slash Swapper is active? Shouldn't be too difficult to implement.

I'm thinking since the functionality of Slash Swapper is so minor (assuming it only swaps `/` and `//`?), adding support for it in this plugin might be more work than simply copying its functionality into this plugin, eliminating the need for Slash Swapper altogether. Not sure if that would be preferable, though.

**Comment by jevonsp** [comment]
I'm not familiar with RL plugin process, would you be able to easily code this in? would really be so helpful

**Comment by YvesW** [comment]
In case you ever want to fix this, this is the slash swapper behaviour based on my tests:
```
slash swapper disabled:
/ = fc
// = cc
/// = guest cc

slash swapper normal:
/ = cc
// = fc
/// = fc (this is likely a bug in slash swapper)

slash swapper with swap guest chat config enabled
/ = guest cc
// = fc
/// = cc
```

You'd then have to use ``PluginManager::isPluginEnabled​`` and of course ConfigManager to get its config. I've not tested what ``isPluginEnabled`` returns if the plugin is not installed, I assume false. The ConfigManager call can ``null`` if the user has never installed the plugin (key doesn't exist) IIRC, so it'd require a null check.

**Comment by ririshi** [comment]
Fixed in 1.5.0! Should appear in plugin hub when the PR gets merged by the kind folks there.
